.github/workflows/ci.yml

```diff
@@ -47,6 +47,13 @@ jobs:
         run: yarn run lint
       - name: Unit Tests
         run: yarn run test --maxWorkers=2
+      - name: Build
+        run: yarn run build
+      - name: e2e tests 🧪
+        uses: cypress-io/github-action@v2
+        with:
+          command: yarn cypress run-ct
+          working-directory: packages/react-urql
```

Comment:
Could be moved to its own file but imo we have some breathing space in terms of execution time, when we add more integrations we should for parallization